docs/react/getting-started.en-US.md

```diff
@@ -0,0 +1,174 @@
+---
+order: 1
+title: Getting Started
+---
+
+Ant Design React is dedicated to providing a **good development experience** for programmers.
+
+---
+
+You had better learn [React](http://facebook.github.io/react/) and [ES2015](http://babeljs.io/docs/learn-es2015/) first.
+
+## First Example
+
+The following CodePen demo is the simplest usage case, and it's also a good habit to fork this demo to provide a re-producible demo while reporting a bug. Please don't use this demo as a scaffold in real world.
+
+- [antd CodePen](http://codepen.io/anon/pen/wGOWGW?editors=001)
+
+## Standard Development Flow
+
+You may need to compile and debug ES2015 and JSX, and proxy and deployment in a project.
+We provide a work flow which is based on `npm` and `webpack`, here is a guide about how to use this work flow.
+
+### 1. Installation
+
+> Please make sure that you had installed [Node.js](https://nodejs.org/en/)(> v4.x) before using `antd-init`.
+
+```bash
+$ npm install antd-init -g
+```
+
+Read [the documentation of `antd-init`](https://github.com/ant-design/antd-init/) and [the documentation of `ant-tool`](http://ant-tool.github.io/) to explore more features.
+
+> Also, you can use scaffold/demo which is provided by community:
+>
+>   - [react-redux-antd](https://github.com/okoala/react-redux-antd)
+>   - [react-antd-admin](https://github.com/fireyy/react-antd-admin)
+>   - [react-antd-redux-router-starter](https://github.com/yuzhouisme/react-antd-redux-router-starter)
+>   - [react-redux-antd-starter](https://github.com/BetaRabbit/react-redux-antd-starter)
+>   - [more](https://github.com/ant-design/ant-design/issues/129)
+
+### 2. Initiate a Project
+
+Initiate a project with cli commands.
+
+```bash
+$ mkdir antd-demo && cd antd-demo
+$ antd-init --type plain-react
+```
+
+`antd-init` will run `npm install` after a project is created. If it fails, you can run `npm install` by yourself.
+
+### 3. Use antd's Components
+
+You can find a Todo App(an valuable demo) in the project which is create by `antd-init`, but it's not important in this guide.
+
+Replace the content of `src/entries/index.js` with the following code.
+As you can see, there is no differeneces between antd's components and usual React components.
```

Comment:
As you can see, there is no difference between antd's components and usual React components.